### ISSUE TITLE
Something broke after 2020-07-03

### ISSUE THREAD

**Issue by dubstard** [gfek/Hunting-New-Registered-Domains#14]
HI,
Based on the zip files i have in the folder of the script, one of the last successfully downloaded ones is 2020-07-03.zip

After this date whenever i try to run the script (i run it for the past day or for 2 days behind) i get an error:
`python hnrd.py -f 2020-09-21 -s paypal
File 2020-09-21.zip does not exist on the remore server.`
I used to see the same error, but in the past it continued on (the zip got downloaded, despite the message saying it i not there)
Now it no longer gets downloaded.
Did something change on the whoisids end?
I still see the files exist, but maybe they now require some sort of subscription?
![image](https://user-images.githubusercontent.com/49607867/94259930-fb24d280-ff37-11ea-9fb8-d76fab0f76f0.png)

Edit:  I WhoisIDs now charge 15 dollars per month for API access. 
Does that imply that the ZIP is no longer accessible for free or they just changed the download links and filenames?

(I can still grab it manually? so i can still use the hnrd.py script:)
The link looks like it is some sort of hash (not a direct link) 

The file arrives as "newly-registered-domains-2020-09-24.zip"
If i rename it to 
`mv newly-registered-domains-2020-09-24.zip 2020-09-24.zip`

and then run the script - it works fine!
`python hnrd.py -f 2020-09-24 -s paypal
`
![image](https://user-images.githubusercontent.com/49607867/94261970-60c68e00-ff3b-11ea-8914-8e3977f167ff.png)

**tl;dr** i suspect whoisids now use a different format for the daily zip than what is in the script:
![image](https://user-images.githubusercontent.com/49607867/94262207-c87cd900-ff3b-11ea-8494-46a7a1c47586.png)

`nrd_zip = 'https://whoisds.com//whois-database/newly-registered-domains/{}/nrd'.format(b64.decode('ascii'))`

![image](https://user-images.githubusercontent.com/49607867/94262939-f7e01580-ff3c-11ea-9e55-18eac7c5c7f6.png)

The file on the server now is a base64 of the date+.zip
and the script only does base64 of the date and manually appends .zip extension
(i am not a coder sorry if my analysis is stupid or wrong)

**Comment by PolarBearGod** [comment]
Crazy enough, I found an article by someone else that ran into this problem. Seems Whoisds is now base64 encoding the date and is using that as the key for zip download. If someone finds time before I do on making a pull to augment this script, this is a blog post that goes over it:

https://www.thatinfosecguy.com/threat-hunting-using-newly-registered-domain-lists/
